zabbix_metrics_watcher.py

```diff
@@ -85,8 +86,12 @@ def send_zabbix_notification(self, zabbix_host, osbs_master, concurrent_builds):
                 zabbix_result[k] = v
             zabbix_result['upload_size_mb'] = self.upload_size_mb
             try:
-                zabbix_result['pulp_push_speed'] =\
-                    self.upload_size_mb / float(zabbix_result['pulp_push'])
+                if 'pulp_push' in zabbix_result.keys():
+                    zabbix_result['pulp_push_speed'] =\
+                        self.upload_size_mb / float(zabbix_result['pulp_push'])
+                elif 'pulp_sync' in zabbix_result.keys():
```

Comment:
Right, 'Pulp upload time' on metrics is the time taken to upload a v1 image. This `else` is added for the builds where v1 is not enabled (we had a couple of those on QA).

After v1 deprecated and no longer used this calculation will be updated to include `tag_and_push` time